Core/Core/Submissions/SubmissionBreakdown.swift

```diff
@@ -112,14 +130,22 @@ struct SubmissionBreakdown: View {
         var body: some View {
             Button(action: action, label: {
                 VStack(spacing: 8) {
-                    CircleProgress(
+                    let circle = CircleProgress(
                         progress: total == 0 ? 0 : CGFloat(count) / CGFloat(total),
                         size: 70,
                         thickness: 7
                     )
-                        .modifier(Counter(count: Double(count)))
-                        .animation(Animation.easeOut(duration: 0.5).delay(0.2))
-                        .padding(.horizontal, 10)
+                    if #available(iOS 14, *) {
```

Comment:
iOS 13.0 crashes when you use an AnimatableModifier. I think it got fixed in a minor version, but I didn't bother to figure out which one, since the ring still animates.